mobile/lib/interfaces/album_api.interface.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:immich_mobile/entities/album.entity.dart';
+
+abstract interface class IAlbumApiRepository {
+  Future<Album> get(String id);
```

Comment:
Do you mind trying to keep the order consistent in all the repos? I usually prefer something like: CRUD, then relations, then misc stuff. Here it would be
- getAll
- create
- update
- delete
- addAssets
- removeAssets
- addUsers
- removeUser